Content.Server/GameObjects/Components/Arcade/RandomArcadeGameComponent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Arcade
+{
+    [RegisterComponent]
+    public class RandomArcadeGameComponent : Component, IMapInit
+    {
+        public override string Name => "RandomArcade";
+
+        public void MapInit()
+        {
+            var arcades = new[]
+            {
+                "BlockGameArcade",
+                "SpaceVillainArcade"
+            };
```

Comment:
Maybe read this from yaml?

Reply:
wouldn't know why tbh. dont know when this should be adjustable